schemes/sima_diagnostics/rayleigh_friction_diagnostics.F90

```diff
@@ -0,0 +1,64 @@
+module rayleigh_friction_diagnostics
+
+   use ccpp_kinds, only:  kind_phys
+
+   implicit none
+   private
+   save
+
+   public :: rayleigh_friction_diagnostics_init ! init routine
+   public :: rayleigh_friction_diagnostics_run  ! main routine
+
+CONTAINS
+
+   !> \section arg_table_rayleigh_friction_diagnostics_init  Argument Table
+   !! \htmlinclude rayleigh_friction_diagnostics_init.html
+   subroutine rayleigh_friction_diagnostics_init(errmsg, errflg)
+      use cam_history,         only: history_add_field
+      use cam_history_support, only: horiz_only
+
+      character(len=512), intent(out) :: errmsg
+      integer,            intent(out) :: errflg
+
+      ! Local variables:
+
+      errmsg = ''
+      errflg = 0
+
+      ! History add field calls
+      call history_add_field('UTEND_RAYLEIGH', 'Zonal wind tendency due to Rayleigh Friction', 'lev', 'avg', 'm s-2')
+      call history_add_field('VTEND_RAYLEIGH', 'Meridional wind tendency due to Rayleigh Friction', 'lev', 'avg', 'm s-2')
+      call history_add_field('STEND_RAYLEIGH', 'Static energy tendency due to Rayleigh Friction', 'lev', 'avg', 'J kg-1')
```

Comment:
I believe this is technically a change in dry air enthalpy as opposed to static energy

```suggestion
      call history_add_field('STEND_RAYLEIGH', 'dry air enthalpy tendency due to Rayleigh Friction', 'lev', 'avg', 'J kg-1')
```

Reply:
Yep! Fixed